extensions/lekab/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lekab",
+  "version": "2.1.0",

```

Comment:
Extensions for Cognigy.AI start with version 4.0.0

Reply:
Thank you, version changed.